### DISCUSSION TITLE
Users unable to unmute and being kicked randomly

### DISCUSSION THREAD

**Discussion by dvdsmthdvd** [mozilla/hubs#6084] [Hubs Cloud Questions]
**Description**

- Using the new template (1.1.16) and users are unable to unmute themselves.
- Separate issue: Users are kicked randomly and see the error attached.

**Additional context**
I've attempted to relaunch my Hubs cloud instance by uploading a new template, but this does not fix the issue. I see no other problems other than the two listed.

![Screenshot_2023-05-16_at_9 40 56_AM](https://github.com/mozilla/hubs/assets/57270645/97d080ed-9f14-40d5-b82b-3ae0a30fe0bc)

**Comment by tortru** [comment]
Same issue but with current 1.1.5 template.
It seems to be a generic issue when starting new servers.

Looks like the new server cannot fetch the latest software packages from bldr.reticulum.io
Therefore the server starts with the old pre-installed versions.

This is a huge problem because this old Reticulum version (1.0.1/20220405231612) is not compatible to the current Hubs client.
It leads to all rooms showing "Voice chat is turned off for this room" and you cannot enable voice chat in the room settings anymore.
